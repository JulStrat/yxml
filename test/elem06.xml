<?xml version="1.0" encoding="UTF-8" ?>
<俄语>данные</俄语>
