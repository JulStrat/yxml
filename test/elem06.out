
elemstart 俄语
content данные
elemend
ok
